lib/RTCSession.js

```diff
@@ -833,7 +835,7 @@ module.exports = class RTCSession extends EventEmitter
 
         if (status_code && (status_code < 200 || status_code >= 700))
         {
-          throw new TypeError(`Invalid status_code: ${status_code}`);
+          return new TypeError(`Invalid status_code: ${status_code}`);
```

Comment:
What?? Sorry, this doesn't make any sense.

Reply:
That new, usually those script kiddies use try catch nothing because they want to ignore those pesky errors the return new Error is a new one for me ahah.

Reply:
Yes, let's assume errors don't exist and write optimistic code. It's fantastic.